docs/source/notebook.md

```diff
@@ -332,6 +332,16 @@ to remember are the following:
 For the full list of available shortcuts, click {guilabel}`Help`,
 {guilabel}`Keyboard Shortcuts` in the notebook menus.
 
+## Searching
+
+Jupyter Notebook has an intelligent built-in search plugin for finding text within a
+notebook or other document, which uses the {kbd}`Ctrl-F` shortcut by default.
+
+Your browser's `find` function will give unexpected results because it doesn't have
+access to the full content of a document, but you can still use your browser find
+function from the browser menu if you want, or you can disable the intelligent search
+shortcut using the advanced settings editor. TODO: DETAILS (investigate settings bug?)
```

Comment:
Should we track this TODO in an isssue?

Reply:
@jtpio I've verified that the settings JSON schema that shows up in the advanced settings editor is different than your sample video. The issue is that I don't know the expected schema for the JSON object being edited, and thus how to explain how to disable the shortcut to a user.

There's also the matter of explaining to users what a JSON object is and what a key/value pair is, on top of where those go in the schema to properly disable the shortcut...this is more content that should be communicated in a docs entry. At this point I'm wondering if simply acknowledging the behaviors of intelligent-search and browser-search (which is already done in the docs in this PR) is enough for now...we can merge this in and worry about docs for disabling later. Thoughts?

Reply:
> simply acknowledging the behaviors of intelligent-search and browser-search (which is already done in the docs in this PR) is enough for now...we can merge this in and worry about docs for disabling later.

Yes, and this is likely something to address in JupyterLab actually.

But it would be nice to not leave the `TODO` in the docs before merging.

Reply:
That's done, I'm opening a new PR for Lab too and will link it here.

Reply:
Same docs updates for JupyterLab are at: https://github.com/jupyterlab/jupyterlab/pull/14667